app/views/support/addresses/edit.html.erb

```diff
@@ -0,0 +1,35 @@
+<%- title = t('page_titles.support.addresses.edit') %>
+<%- content_for :title, title_with_error_prefix(title, @school.errors.any?) %>
+<%- content_for :before_content do %>
+<%- end %>
+
+<div class="govuk-grid-row">
```

Comment:
Can we add a `Back` link to the top of the page, which returns the user to the school detail page they came from